src/lib/drupal/facilityNoDrupalPageSideNav.ts

```diff
@@ -0,0 +1,23 @@
+import { SidebarData } from '@/types/formatted/staffProfile'
```

Comment:
If this is going to be a utility in `src/lib/drupal/`, it shouldn't rely on a `staffProfile` type. Can we extract this `SidebarData` into a new module?

Reply:
I renamed this because it is only related to staff profile the confusing name is from content-build and its only used in profiles over there.